src/NotesLoaderSSC.cpp

```diff
@@ -499,6 +503,9 @@ void SetStepsAttacks(StepsTagInfo& info)
 		info.loader->ProcessAttacks(info.steps->m_Attacks, *info.params);
 	}
 }
+// steps offset/sync bias actually does not make any sense for me
+// there is single audio file and it has single offset
+// implement this anyway
```

Comment:
So, the problem we're trying to tackle here is when a stepfile's offset value has had 9ms added _after it was synced_ if the author wanted it to have ITG offset.

Don't think about the audio file at all here, what we want to do is detect if the pack is marked as ITG sync, and if so, subtract 9ms from the offset value we read from the file before we give it to the engine.

This way, everything is treated inside the engine as if it's null sync.